ui/widgets/playercontrols.go

```diff
@@ -10,6 +10,10 @@ import (
 	"fyne.io/fyne/v2/widget"
 )
 
+var (
+	themedResReplay = theme.NewThemedResource(theme.MediaReplayIcon())
```

Comment:
the icons that come built into Fyne are already ThemedResources, you can just use `theme.MediaReplayIcon()` directly to create the button

Reply:
This was my first approach. However, I found that those built-in icons are globals for the entire app ([icons is a map in the global scope](https://github.com/fyne-io/fyne/blob/1834d704a305f4729cce65213f5b5c3098e1d993/theme/icons.go#L456)). That means that, when changing the icon color, it would be affecting any other usages of that same icon within the application.

For this reason, as we are manipulating the `ThemedResource.ColorName` attribute, I preferred to create new objects to be used just for this new button, and avoid being bitten by global state changes.

Reply:
Yeah this makes sense since you're changing the icon color. I missed that the first time around. If we stick with changing the icon color (see other comment at line 155) then we need this, and will need it for the album filters button if I didn't already create a local themedResource (don't know off the top of my head).